### PR TITLE
Update tech radar docs on using an external datasource

### PR DESCRIPTION
**Pull request by joosthofman**
Signed-off-by: Joost Hofman <joost.hofman@ah.nl>

Added extra documentation on how to load an external datasource for the tech radar properly.
A convert is needed when working with Date objects in the json datasource.

#### :heavy_check_mark: Checklist

- [x] Added or updated documentation
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/tech-radar/README.md [hunk 1]
```diff
@@ -91,8 +122,8 @@ export class MyOwnClient implements TechRadarApi {
     const data = await fetch('https://mydata.json').then(res => res.json());
 
     // maybe you'll need to do some data transformation here to make it look like TechRadarLoaderResponse
-
-    return data;
+    // Need a converter for the Date object
+    return convertTechRadarData(data);
```
Comment (Rugvip) [inline plugins/tech-radar/README.md:126]:
```suggestion
    // For example, this converts the timeline dates into date objets
    return {
      ...data,
      entries: data.entries.map(entry => ({
        ...entry,
        timeline: entry.timeline.map(timeline => ({
          ...timeline,
          date: new Date(timeline.date),
        }))
      }))
    }
```

**Review by Rugvip — CHANGES_REQUESTED** [review]
Thanks! makes sense to call that out 👍 

What do you think about this slightly slimmer option?

File: plugins/tech-radar/README.md (hunk 1 above)
Comment (joosthofman) [reply plugins/tech-radar/README.md]: That's even a better solution :)
Updating PR.

**Review by Rugvip — APPROVED** [review]
Nice, let's :shipit: !

**Pull request merged**